src/test/java/example/Example.java

```diff
@@ -86,5 +94,21 @@ public void initialize(Bootstrap<ExampleConfiguration> bootstrap) {
         public void run(ExampleConfiguration configuration, Environment environment) throws Exception {
             environment.jersey().register(new ExampleResource());
         }
+
+        @VisibleForTesting
```

Comment:
but you are calling getWebSecurityBundle from ExampleRestApplication though... i dont think that is not a good example :(